Openssh default configurstion disallows root login.
Fix this by patching the config file
Upstream: n/a
Signed-off-by: Andreas Ziegler <br025@umbiko.net>
---
[Rebase for version 10]
Signed-off-by: Andreas Ziegler <br025@umbiko.net>
---
--- a/openssh-10.0p1/sshd_config     2025-04-09 09:02:43.000000000 +0200
+++ b/sshd_config       2025-09-25 10:23:55.329450086 +0200
@@ -29,7 +29,7 @@
 # Authentication:
 
 #LoginGraceTime 2m
-#PermitRootLogin prohibit-password
+PermitRootLogin yes
 #StrictModes yes
 #MaxAuthTries 6
 #MaxSessions 10
